spec/anagrams_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'anagrams'
+
+describe Anagrams do
+  describe '.to_s' do
+    it "returns all anagrams from a file" do
+      an = Anagrams.new("test.txt")
```

Comment:
@StefanGajic it would be much easier to test `Anagrams` if it was receiving an array of words, a.k.a. dictionary. This would also make it easier to change input, e.g. use it for a web-based input. Please modify it to receive an array of words instead... even better make it so if a string is provided it treats is as a path to a file, but if an array is given it should treat it as a word list.